jujugui/static/gui/src/app/components/profile/profile.js

```diff
@@ -191,28 +188,25 @@ class Profile extends React.Component {
     }
 
     return (
-      <div>
-        <Panel
-          instanceName="profile"
-          visible={true}>
-          <ProfileHeader
-            changeState={props.changeState}
-            controllerIP={props.controllerIP}
-            getUser={props.getUser}
-            gisf={props.gisf}
-            userInfo={shapeup.fromShape(props.userInfo, ProfileHeader.propTypes.userInfo)} />
-          <div className="twelve-col">
-            <div className="profile__content inner-wrapper">
-              <ProfileNavigation
-                // Use supplied activeSection or the key from the first map entry.
-                activeSection={this._getSectionInfo().active || mapEntry[0]}
-                changeState={props.changeState}
-                sectionsMap={sectionsMap} />
-              {section.getComponent()}
-            </div>
+      <Panel instanceName="profile"
```

Comment:
The inconsistency where sometimes an attribute it on the same line as the tag name and sometimes not actually makes things harder to read.